azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageView.kt

```diff
@@ -78,11 +78,17 @@ internal fun MessageView(viewModel: MessageViewModel, dispatch: Dispatch) {
                 stringResource = R.string.azure_communication_ui_chat_joined_chat,
                 substitution = viewModel.message.participants
             )
-            ChatMessageType.PARTICIPANT_REMOVED -> SystemMessage(
-                icon = R.drawable.azure_communication_ui_chat_ic_participant_removed_filled,
-                stringResource = R.string.azure_communication_ui_chat_left_chat,
-                substitution = viewModel.message.participants
-            )
+            ChatMessageType.PARTICIPANT_REMOVED -> if (viewModel.message.isCurrentUser)
```

Comment:
We should expect that in one event from server side we might receive a list of removed users that has local participant plus remote participants.
We should have posted two messages